src/util/validator/CourseValidator.tsx

```diff
@@ -0,0 +1,31 @@
+import { Author } from 'src/components/Courses/components/CourseCard/Author.types';
+import { Course } from 'src/course.type';
+
+export const isNewCourseValid = (newCourse: Course) => {
+	return (
+		isCourseTitleValid(newCourse.title) &&
+		isCourseDescriptionValid(newCourse.description) &&
+		isCourseDurationValid(newCourse.duration) &&
+		newCourse.authors.length > 0
+	);
+};
+
+export const isCourseTitleValid = (title: string) => {
+	return title.length >= 2;
+};
+
+export const isCourseDescriptionValid = (description: string) => {
+	return description.length >= 2;
+};
+
+export const isCourseDurationValid = (duration: number) => {
+	return duration > 0;
+};
+
+export const isAuthorNameValid = (description: string) => {
+	return description.length >= 2;
+};
+
+export const isCourseAuthorListValid = (courseAuthorList: Author[]) => {
```

Comment:
there a lot of very similar functions, you can create for example something like: isInputHasValidLength
```
const isInputHasValidLength = (input: string, length: number) => (
  input.length >= length
)
```

Reply:
in the future some validation logic might be added for the field and I can add some logic  only in validation method without changes in other files